modules/internal/src/main/scala/higherkindness/mu/rpc/internal/serviceImpl.scala

```diff
@@ -441,7 +441,7 @@ object serviceImpl {
 
         val executionClient: Tree = response match {
           case MonixObservableTpe(_, _) =>
-            q"client.stream(request).flatMap(_.asStream[${response.safeInner}]).toObservable"
+            q"Observable.fromReactivePublisher(client.stream(request).flatMap(_.asStream[${response.safeInner}]).toUnicastPublisher)"
```

Comment:
We need fqn for types: `_root_.monix.reactive.Observable`

The same would apply for the rest of the usages in this file.

Reply:
Done at https://github.com/higherkindness/mu/pull/552/commits/9b0a8f3c29fe1b00b30c08db8cb695f5bbdaf252